build.sbt

```diff
@@ -48,26 +50,32 @@ lazy val core = project
   .settings(moduleName := "snowplow-event-recovery")
   .settings(buildSettings)
   .settings(
-    resolvers += "Snowplow Analytics Maven repo" at "http://maven.snplow.com/releases/",
+    resolvers ++= Seq(
+      "Snowplow Analytics Maven repo" at "http://maven.snplow.com/releases/",
+      "Twitter Maven repo" at "https://maven.twttr.com/"
+    ),
     libraryDependencies ++= Seq(
       "com.snowplowanalytics" % "collector-payload-1" % thriftSchemaVersion,
       "org.typelevel" %% "cats-core" % catsVersion,
       "io.lemonlabs" %% "scala-uri" % scalaUriVersion,
       "io.circe" %% "circe-optics" % circeOpticsVersion,
       "com.snowplowanalytics" %% "iglu-scala-client" % igluClientVersion,
+      "com.snowplowanalytics" %% "snowplow-badrows" % badRowsVersion,
       "org.scalatest" %% "scalatest" % scalatestVersion % "test",
       "org.scalacheck" %% "scalacheck" % scalacheckVersion % "test",
       "com.snowplowanalytics" %% "scalacheck-schema" % scalacheckSchemaVersion % "test",
       ("com.snowplowanalytics" %% "snowplow-common-enrich" % sceVersion % "test")
```

Comment:
Do we still need this dependency now that bad rows have been put into their own library?